setup.py

```diff
@@ -40,9 +40,9 @@
       packages=find_packages(exclude=('tests','*.tests')),
       install_requires=[
         "click==6.7",
-        "PyYAML==4.2b1",
+        "PyYAML>=5.0",
```

Comment:
We can probably remove this because pip will pull in the right requirement from `oyaml`.